themes/custom/az_barrio/css/style.css

```diff
@@ -47,6 +47,38 @@ audio:not([controls]) {
   display: none;
 }
 
+.navbar-offcanvas {
```

Comment:
This can be removed if we merge https://github.com/az-digital/arizona-bootstrap/pull/314